### ISSUE TITLE
(Bug) Left button drag default "window move" option is broken on X11

### ISSUE THREAD

**Issue by Andrew-Mattias** [antroids/application-title-bar#22]
Easily reproducible on X11 session of Plasma 6. In Wayland session everything is fine and working as intended.

![изображение](https://github.com/antroids/application-title-bar/assets/91881068/8913f885-97ac-4ca4-b560-760995d2c7d9)

Possible easy solution - deactivate this option in ComboBox and rename in as "Window Move <unavailable on X11>" if X11 Plasma session is detected, because will be X11 phased out soon anyway. 

It will be a bit sad though, because I have to use X11 Plasma to remote into my PC through AnyDesk, which allows remoting only into X11 sessions.

I tried a couple of left button drag options, some of them are fine on X11, some are broken. I would elaborate on this matter later in the comments for this issue.

**Comment by antroids** [comment]
I added a general warning and error messages for Drag => Window Move action.
Looks like x11 cannot handle move action properly when mouse button already pressed.
Probably, there are some other unsupported combinations, but I don't want to test them.

**Comment by Andrew-Mattias** [comment]
> I added a general warning and error messages for Drag => Window Move action. Looks like x11 cannot handle move action properly when mouse button already pressed. Probably, there are some other unsupported combinations, but I don't want to test them all.

Yes, there are plenty of bugs related to mouse dragging when X11 session is used. I have documented only "window move one" because it is the option set as a default and i use it extensively. I am planning to play more with mouse drag feature so I will file bug reports on this matter as I encounter them.
